### PR TITLE
port Generating container recipes & images page to Markdown

### PR DESCRIPTION
**Pull request by orbsmiv**

### CODE REVIEW

**Comment by orbsmiv** [comment]
Need to update internal headers, moving the --commands to the next line, where required

File: docs/containers.md [hunk 1]
```diff
@@ -1,712 +1,721 @@
...
+##### Requirements for base container image {: #containers_usage_container_base_image_requirements }
 
-Requirements for base container image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are a couple of specific requirements for the base container
+image:
 
-There are a couple of specific requirements for the base container image:
+- all dependencies of EasyBuild must be installed, including:
+    - Python 2.6 or 2.7
```
Comment (branfosj) [inline docs/containers.md:171]:
```suggestion
    - Python 2.7 or 3.6+
```

File: docs/containers.md [hunk 2]
```diff
@@ -1,712 +1,721 @@
...
+recipes & images, use the `--containerpath` configuration setting. Next
+to providing this as an option to the `eb` command, you can also define
+the `$EASYBUILD_CONTAINERPATH` environment variable or specify
+`containerpath` in an EasyBuild configuration file.
 
-See also https://www.sylabs.io/guides/latest/user-guide/build_a_container.html .
+The default value for this location is
+`$HOME/.local/easybuild/containers`, unless the `--prefix` configuration
+setting was provided, in which case it becomes `<prefix>/containers`
+(see [Overall prefix path][prefix].
```
Comment (branfosj) [inline docs/containers.md:567]:
```suggestion
(see [Overall prefix path][prefix]).
```

File: docs/containers.md [hunk 3]
```diff
@@ -1,712 +1,721 @@
...
+    the value for `--container-image-format` is ignored when creating Docker
+    container images.
 
-You can specify an altername name using the ``--container-image-name`` configuration setting.
+The format for container images that EasyBuild is produces via the
+functionality provided by the container software can be controlled via
+the `--container-image-format` configuration setting.
 
-The filename of generated container recipe will be ``Singularity.<name>``.
+For Singularity containers (see [Type of container recipe/image to generate][containers_cfg_type])),
```
Comment (branfosj) [inline docs/containers.md:585]:
```suggestion
For Singularity containers (see [Type of container recipe/image to generate][containers_cfg_type]),
```

**Review by branfosj — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]

**Pull request merged**